tests/test_restore_names.py

```diff
@@ -1,3 +1,86 @@
 from app.restore_names import restore_names
 
-# write your tests here
+
+def test_when_first_name_did_not_lose():
+    users = [
+        {
+            "first_name": "Sheldon",
+            "last_name": "Cooper",
+            "full_name": "Sheldon Cooper",
+        },
+        {
+            "first_name": "Leonard",
+            "last_name": "Leonard",
```

Comment:
"first_name": "Leonard", == "last_name": "Leonard",
but
"full_name": "Leonard Hofstadter",
How it's possible?